core/Pos/Core/Constants/Raw.hs

```diff
@@ -86,8 +86,9 @@ data CoreConfig = CoreConfig
       ccGenesisN                     :: !Int
       -- | Size of mem pool will be limited by this value muliplied by block
       -- size limit.
-    , ccMemPoolLimitRatio            :: !Word
-      -- | Suffix for genesis.bin files
+    , ccMemPoolLimit                 :: !Int
+      -- | Limint on the number of transactions that can be stored in
+      -- the mem pool.
```

Comment:
You've edited the wrong comment here. The `-- |` style comments precede the things they're documenting.

Reply:
Oh no! I'll correct that.